aries-site/src/examples/components/avatar/AvatarDataTableExampleDo.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Avatar, Box, DataTable, ResponsiveContext, Text } from 'grommet';
+
+const data = [
+  {
+    name: 'Kenny Tran',
+    avatar: {
+      src: '/Kenny.png',
+    },
+    status: 'Verified',
+    platformRole: 'Admin',
+    lastActive: '4 hours ago',
+  },
+  {
+    name: 'Vicky Avalos Campos',
+    avatar: {
+      src: '/Vicky.png',
+    },
+    status: 'Invite Pending',
```

Comment:
```suggestion
    status: 'Invite pending',
```
Think these statuses should all be sentence cased -- will need to adjust for all instances.